common/src/main/resources/assets/voicechat/lang/pt_br.json

```diff
@@ -1,25 +1,25 @@
 {
   "key.categories.voicechat": "Voice Chat",
-  "key.push_to_talk": "Mantenha pressionado para falar",
+  "key.push_to_talk": "Mantenha pressionado p/ falar",
   "key.whisper": "Sussurrar",
   "key.mute_microphone": "Silenciar Microfone",
   "key.disable_voice_chat": "Desabilitar Voice Chat",
   "key.hide_icons": "Ocultar ícones do Voice Chat",
-  "key.voice_chat_settings": "Configurações",
+  "key.voice_chat_settings": "Configs.",
```

Comment:
Why did you change this?

Reply:
Simplifies and abbreviates the word, without losing the meaning of the word in Portuguese.

Reply:
Is there any reason to do this? Is the translation too wide to be displayed correctly in game?

Reply:
This helps to make the text smaller, which makes the buttons prettier and doesn't lose meaning in translation.

Reply:
This is subjective. As long as there is enough space for translations in the game, there is no need to do this.

Reply:
Do you think it's better to reverse it and leave it without the abbreviation then?

Reply:
Yes